config/initializers/simple_form.rb

```diff
@@ -1,5 +1,6 @@
 # Use this setup block to configure all options available in SimpleForm.
 SimpleForm.setup do |config|
+  config.input_class = "form-control"
```

Comment:
This option should be redundant because it's part of the `:vertical_form` wrapper defined in simple_form_bootstrap.rb, which should in turn be the default wrapper. Did you find that adding it actually changes the resulting HTML?